cmd/acra-server/acra-server.go

```diff
@@ -253,35 +253,35 @@ func main() {
 		}
 
 		var ocspClientConfig *network.OCSPConfig
-		if len(*tlsOcspClientUrl) > 0 {
-			ocspClientConfig, err = network.NewOCSPConfig(*tlsOcspClientUrl, *tlsOcspRequired, *tlsOcspFromCert)
+		if *tlsOcspClientURL != "" {
```

Comment:
@Lagovas, I'm afraid I don't quite follow you. Could you please elaborate with a practical example?

Let's take OCSP URL for example. I can see three possible parameters here:

1. `tls_ocsp_url` — set URL for both client and DB
2. `tls_client_ocsp_url` — set URL only for certificates used by the client
3. `tls_db_ocsp_url` — set URL only for certificates used by the DB

which can be combined in the following ways, as I see it:

1. Provide only one: `tls_ocsp_url`.
    - Common configuration, only one OCSP authority.
2. Use just two: `tls_client_ocsp_url` and  `tls_db_ocsp_url`.
    - Distinct configurations possible, may use different OCSP servers.
    - If configurations are the same, specify both with the same value.
    - Easily extendable for more configurations if needed.
3. All three: `tls_ocsp_url` + `tls_client_ocsp_url` & `tls_db_ocsp_url`.
    - Distinct configurations possible, may use different OCSP servers if needed.
    - If the configuration is the same, specify just common `tls_ocsp_url`.
    - Easily extendable for more configurations if needed.

Which one do you suggest to implement? If there's a fourth option, please note it as well.

I personally prefer the option 3 as it can be convenient for easy configuration when there is only one authority, but can be easily used in advanced mode to override configuration for client/DB if necessary. However, it also introduces some complexity so the option 2 is quite viable as well (albeit not so convenient on the command-line, for example). Option 1 is inconsistent with the rest of the TLS configuration now, so I wouldn't consider it.

Reply:
I thought that enough only 2 case with 2 separate values. it not hard to copy paste same value for both parameters. but... probably u r right and we can simplify configuration for simple configurations with one ocsp/crl parameter for both sides